### PR TITLE
docs: adds context for docs

### PR DESCRIPTION
**Pull request by rstavchansky**
Added mentions of Essential Next.js plugin for users coming to these docs pages from links outside this repo.
Created this PR in tandem with an open Netlify docs PR. (https://github.com/netlify/docs/pull/1235)

### CODE REVIEW

File: docs/custom-redirects.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Custom Netlify Redirects
 
-You can define custom redirects in a `_redirects` file.
+You can define custom redirects in a `_redirects` file for apps using the Essential Next.js plugin.
+
 The precedence of these rules are:
 
 - `_redirects`
```
Comment (rstavchansky) [inline docs/custom-redirects.md:7]: Just checking on the line below whether this should still read "`next-on-netlify` redirects" or whether it needs to be updated.

Comment (lindsaylevine) [reply docs/custom-redirects.md:7]: oooo yes you could update the line below to " `Essential Next.js` redirects " 💡

**Review by lindsaylevine — APPROVED** [review]
thanks for this!!!!!!!!!!!!!!!

**Comment by lindsaylevine** [comment]
commit name needs to be appended with either chore fix or feat so `chore: adds context for docs`. lmk if u need help updating/rebasing :D

**Pull request merged**